packages/react-components/react-positioning/src/createSlideStyles.ts

```diff
@@ -8,12 +8,13 @@ import { DATA_POSITIONING_PLACEMENT } from './constants';
  * @returns Griffel styles to spread to a slot
  */
 export function createSlideStyles(mainAxis: number): GriffelStyle {
+  // With 'accumulate' animation composition, the opacity keyframes are deltas from the default value of 1.
   const fadeIn = {
     from: {
-      opacity: 0,
+      opacity: -1, // default of 1 - 1 = 0 actual
     },
     to: {
-      opacity: 1,
+      opacity: 0, // default of 1 + 0 = 1 actual
```

Comment:
Let's do undo this, this is a very tricky and fragile fix 💥 

As we have two separate motions, let's allow the browser to do its work:

```diff
-   animationComposition: 'accumulate',
+   animationComposition: 'replace, accumulate',
    animationDuration: tokens.durationSlower,

    // Note: at-rules have more specificity in Griffel
    '@media(prefers-reduced-motion)': {
      [`&[${DATA_POSITIONING_PLACEMENT}]`]: {
+       animationComposition: 'replace',
        animationDuration: '1ms',
        animationName: fadeIn,
      },
    },
    // Tested in Firefox 79
    '@supports not (animation-composition: accumulate)': {
      [`&[${DATA_POSITIONING_PLACEMENT}]`]: {
+       animationComposition: 'replace',
        animationName: fadeIn,
      },
    },
  };
```

Reply:
@layershifter Good suggestion to separate the animation composition modes; done.